dashboard/src/redux/reducers/toaster.reducer.ts

```diff
@@ -0,0 +1,66 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from '@/redux/store';
+
+export type ToasterState = {
+    message: string;
+    duration: number;
+    type: string;
+    id: number;
+    status: string;
+};
+
+export const ToasterType = {
```

Comment:
如果是不變的常數，建議可以用 `全大寫_全大寫_...` 的 constant naming 方式命名 (如 constants folder 中的概念)

```suggestion
export const TOASTER_TYPE = {
```